app/views/courses/_cohort_courses.html.erb

```diff
@@ -17,7 +17,8 @@
     </div>
     <div class="cohort-select">
       <select class="cohorts">
-        <option selected disabled><%= t("cohort.view_other") %></option>
+        <option selected disabled>
+        <%= Features.wiki_ed? ? t("courses.view_other") : t("courses_generic.view_other") %></option>
```

Comment:
It will be available in all the rails view templates, whether erb or haml. It won't be available in the jsx.